proposals/devcontainer-features.md

```diff
@@ -90,13 +108,12 @@ In most cases, the `devcontainer-collection.json` file can be generated automati
 
 ## devcontainer.json properties
 
-Features are referenced in `devcontainer.json` , where the `features` tag consists of an array with the ordered list of features to be included in the container image.
+Features are referenced in `devcontainer.json` , where the `features` tag consists of an object tag starting with the id of the feature and including the values of the options to pass to the feature itself.
 
 The properties are:
 | Property | Type | Description |
 | :--- | :--- | :--- |
-| id | string | Reference to the particular feature to be included. |
-| options | object | Type of the option .|
+| options | object | Type of the option. |
```

Comment:
We don't need a separate property for options.

Reply:
Removed.